src/routes/docs/tutorials/nuxt/step-4/+page.markdoc

```diff
@@ -0,0 +1,196 @@
+Similarily, if you type something in the input field, the information in your code also updates.
+It is a two-way-cnnection that are magically links them together.
+
+Create the form by adding a new file `src/pages/index.vue` and put in the following code.
+
+```vue
+<template>
+  <div class="card u-margin-32">
+    <h2 class="eyebrow-heading-2">Login/Register</h2>
+    <form
+      class="form u-width-full-line u-max-width-500 u-margin-block-start-16"
+      @submit.prevent="handleLogin || handleRegistration"
+    >
+      <ul class="form-list">
+        <li class="form-item">
+          <label class="label">Email</label>
+          <div class="input-text-wrapper">
+
+            <!-- Input with v-model to link email field to script -->
+            <input
+              v-model="userData.email"
+              type="email"
+              class="input-text"
+              placeholder="Email"
+              required
+            />
+          </div>
+        </li>
+        <li class="form-item">
+          <label class="label">Password</label>
+          <div class="input-text-wrapper">
+
+            <!-- Input with v-model to link password field to script -->
+            <input
+              v-model="userData.password"
+              type="password"
+              class="input-text"
+              placeholder="Password"
+              required
+            />
+          </div>
+        </li>
+      </ul>
+      <ul class="buttons-list u-margin-block-start-16">
+        <li class="buttons-list-item">
+
+        <!-- Login button -->
+          <button
+            class="button is-small u-margin-inline-start-4"
+            aria-label="Login"
+            @click="handleLogin"
+          >
+            Login
+          </button>
+        </li>
+        <li class="buttons-list-item">
+
+        <!-- Register button -->
+          <button
+            class="button is-small is-secondary u-margin-inline-start-4"
+            aria-label="Register account"
+            @click="handleRegistration"
+          >
+            Register
+          </button>
+        </li>
+      </ul>
+    </form>
+  </div>
+</template>
+
+<script>
+export default {
+  setup() {
+    //Accessing the user session composable
+    const user = useUserSession();
+
+    //V-model object
+    const userData = {
+      email: "",
+      password: "",
+    };
+
+    //Event handler for logging in
+    const handleLogin = async () => {
+      await user.login(userData.email, userData.password);
+    };
+
+    //Event handler for registrering
+    const handleRegistration = async () => {
+      await user.register(userData.email, userData.password);
+    };
+
+    return {
+      handleLogin,
+      handleRegistration,
+      userData,
+    };
+  },
+};
+</script>
+```
+
+# Home page {% #home-page %}
+
+Now that users can register and login to the app, let's modify the home page in `src/pages/index.vue` to give the logged in users a secret message.
```

Comment:
Didn't we just change this in the last step??

Are the files you refer to correct?